nginx/nginx.conf

```diff
@@ -10,18 +10,52 @@ http {
         ~image/                    max;
     }
 
-server {
-    listen 80;
-    server_name _;
-    expires $expires;
-    index index.html;
-    root /usr/share/nginx/html;
-    include /etc/nginx/mime.types;
+    server {
+        listen 80;
+        server_name _;
+        expires $expires;
+        index index.html;
+        root /usr/share/nginx/html;
+        include /etc/nginx/mime.types;
 
-    location / {
+        # redirect if someone tries to open in http
+        # if ($http_x_forwarded_proto = 'http') {
+        #   return 301 https://$host$request_uri;
+        # }
+
+        # X-Frame-Options is to prevent from clickJacking attack
+        add_header X-Frame-Options SAMEORIGIN;
+        
+        # disable content-type sniffing on some browsers.
+        add_header X-Content-Type-Options nosniff;
+        
+        # This header enables the Cross-site scripting (XSS) filter
+        add_header X-XSS-Protection "1; mode=block";
+        
+        # This will enforce HTTP browsing into HTTPS and avoid ssl stripping attack
+        add_header Strict-Transport-Security "max-age=31536000; includeSubdomains;";
+        
+        add_header Referrer-Policy "no-referrer-when-downgrade";
+        
+        # Enables response header of "Vary: Accept-Encoding"
+        gzip on; 
+        # gzip_vary on;
+
+        location / {
             try_files $uri $uri/ /index.html;
             add_header Cache-Control "no-store, no-cache, must-revalidate";
         }
-    }
 
+        # Media: images, icons, video, audio, HTC
+        location ~* \.(?:jpg|jpeg|gif|png|ico|cur|gz|svg|svgz|mp4|ogg|ogv|webm|htc)$ {
+            # expires 1M;
+            add_header Cache-Control "public";
+        }
```

Comment:
is this going to cause it to crash again?

Reply:
So these location tags are now inside server { } tag. they were outside before which was the error Liban posted the error.